### PR TITLE
JAMES-2456 Add Tika information in README & docker-compose

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -221,6 +221,10 @@ You need a running *ElasticSearch* in docker. To achieve this run :
 
     $ docker run -d --name=elasticsearch elasticsearch:2.2.1
 
+If you want to use all the search JMAP capabilities, you may also need to start Tika :
```
Comment (rouazana) [inline README.adoc]: s/search JMAP/JMAP search/

Comment (rouazana) [reply README.adoc]: s/ :/:

Comment (rouazana) [reply README.adoc]: maybe you can explain why this specific container is needed

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**